<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1271</width>
      <height>851</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/back/BACKGROUND.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="backPB">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>550</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(170, 170, 127)</string>
     </property>
     <property name="text">
      <string>Back to Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EditTTPB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>290</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(170, 170, 127)</string>
     </property>
     <property name="text">
      <string>Edit Timetable</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MarkLeavePB">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>250</y>
       <width>290</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(170, 170, 127)</string>
     </property>
     <property name="text">
      <string>Mark Leave</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="r4.qrc"/>
  <include location="r4.qrc"/>
 </resources>
 <connections/>
</ui>
